Does not work with multi dot names anymore

After https://github.com/ember-template-lint/ember-template-lint/pull/3133 (released in 7.0.1), it is not possible to lint files with multi dot name like `.test.gjs`

Should be the multi dot names supported out of the box?

Thanks

Thanks @lifeart, but using v7.0.8 ends with same error like

```
./node_modules/ember-template-lint/bin/ember-template-lint.js app/app/helpers/feature.test.gjs
app/app/helpers/feature.test.gjs .test.gjs .gjs
Linting 1 Total Files with TemplateLint
        .test.gjs: 1

app/app/helpers/feature.test.gjs .test.gjs .gjs
./fallback-or-is-from-stdin.hbs .hbs .hbs
app/app/helpers/feature.test.gjs
  1:0  error  Templates are required to be in strict mode. Consider refactoring to template tag format.
```

It looks that [canProcessFile](https://github.com/ember-template-lint/ember-template-lint/blob/master/lib/-private/file-path.js#L29) returns `false` for `app/app/helpers/feature.test.gjs`, hence using `.hbs` as a fallback 🤔

--- a/lib/-private/file-path.js
+++ b/lib/-private/file-path.js
@@ -31,9 +31,13 @@ export function canProcessFile(filePath) {
     return false;
   }
 
-  const { ext } = parseFilePath(filePath);
+  const { ext, shortExt } = parseFilePath(filePath);
 
-  return isDTS(ext) || isGlimmerScript(ext) || isHBS(ext) || isHTML(ext) || isScript(ext);
+  if (isDTS(ext)) {
+    return true;
+  }
+
+  return isGlimmerScript(shortExt) || isHBS(shortExt) || isHTML(shortExt) || isScript(shortExt);
 }
 
 /**

--- a/test/unit/-private/file-path-test.js
+++ b/test/unit/-private/file-path-test.js
@@ -69,13 +69,17 @@ describe('file-path', function () {
       // Glimmer script files
       expect(canProcessFile('component.gjs')).toBe(true);
       expect(canProcessFile('component.gts')).toBe(true);
+      expect(canProcessFile('app/app/helpers/feature.test.gjs')).toBe(true);
 
       // HTML files
       expect(canProcessFile('template.html')).toBe(true);
+      expect(canProcessFile('template.test.html')).toBe(true);
 
       // JavaScript and TypeScript files (soon deprecated)
       expect(canProcessFile('script.js')).toBe(true);
+      expect(canProcessFile('script.test.js')).toBe(true);
       expect(canProcessFile('script.ts')).toBe(true);
+      expect(canProcessFile('script.test.ts')).toBe(true);
 
       // TypeScript declaration files
       expect(canProcessFile('types.d.ts')).toBe(true);

